tests/test_emails.py

```diff
@@ -91,6 +91,52 @@ def test_send_emails(
     email_connection.get_email_message(to=recipients, **expected_call_kwargs)
 
 
+@mock.patch("saleor.order.emails.send_templated_mail")
+def test_send_staff_emails_without_notification_recipient(
+    mocked_templated_email, order, site_settings
+):
+    emails.send_staff_order_confirmation(order.pk)
+    email_data = emails.collect_staff_data_for_email(
+        order.pk, emails.STAFF_CONFIRM_ORDER_TEMPLATE
+    )
+
+    expected_call_kwargs = {
+        "context": email_data["context"],
+        "from_email": site_settings.default_from_email,
+        "template_name": emails.STAFF_CONFIRM_ORDER_TEMPLATE,
+    }
+
+    mocked_templated_email.assert_called_once_with(
+        recipient_list=[], **expected_call_kwargs
+    )
```

Comment:
Is it supposed to be called if there are no recipient?

Reply:
fixed